document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreQueryV1Test.java

```diff
@@ -349,6 +350,178 @@ public void testFindWithNestedFields(String dataStoreName) throws IOException {
     testCountApi(dataStoreName, query, "query/filter_on_nested_fields_response.json");
   }
 
+  @ParameterizedTest
+  @MethodSource("databaseContextPostgres")
```

Comment:
All the tests should be for both MongoDB and Postgres so that we have compatibility.
```suggestion
  @MethodSource("databaseContextBoth")
```

Reply:
The id being returned in case of mongo is different from what is being returned from postgres. I had discussion with @kotharironak, and he said it will be fixed later.

Reply:
Sure.